src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -607,5 +607,26 @@
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
 		}
+//		Disabled until it gets fixed (being sent at least 3 times on both Paper and Spigot)
```

Comment:
Paper has marked this as expected behaviour since it's how the inventory was designed to interact with the client, so it won't be 'fixed'.

> As has already been stated, this is expected behavior; Outside of rewriting the entirety of how anvils work this is basically unavoidable

_Originally posted by @electronicboy in https://github.com/PaperMC/Paper/issues/4844#issuecomment-1047067975_

You can choose to add it now in its current state or not add it at all, but from what I can tell there is no plan to change the unusual behaviour.